xml/docu_styleguide.structure.xml

```diff
@@ -2667,10 +2667,77 @@ xml:id="tab-install-source"</screen>
    <tag class="attribute">xml:id</tag>
    attribute.
   </para>
-  <para>
-   Value and description pairs are better handled by means of a descriptive
-   list.
-  </para>
+
+  <para>Typical usecases of tables include:</para>
+  
+  <variablelist>
+   <varlistentry><term>Lookup tables for specific information</term>
+    <listitem>
+     <para>Whenever users need to check for data that applies to them, 
+      create lookup tables. For example, use a table to sum up system requirements.
+      </para>
+    </listitem>
+   </varlistentry>
+   <varlistentry><term>Matrix tables</term>
+    <listitem>
+     <para>Whenever users need to quickly check whether a specific combination
+      of options work or not. For example, use a matrix table to visualize supported 
+      combinations or update options.</para>
+    </listitem>
+   </varlistentry>
+  </variablelist>
+
+  <para>As there are use cases for tables, there are cases when <emphasis>not</emphasis> 
+   to use tables:</para>
+  
+  <itemizedlist>
+   <listitem>
+    <para>Value and description pairs are better handled by means of a descriptive list.</para>
+   </listitem>
+   <listitem>
+    <para>Wordy explanations or descriptions should not be used in tables.</para>
+   </listitem>
+   <listitem>
+    <para>Complex layout constructs (screens, code) should not be used in tables.</para>
+   </listitem>
+  </itemizedlist>
+
+  <para>Some styling/style tips for tables:</para>
+  
+  <variablelist>
+   <varlistentry><term>Structure your table to have more rows than columns</term>
+    <listitem>
+     <para>Humans prefer to skim for information that is aligned vertically. When designing 
```

Comment:
'Humans' stands a bit out here :-) Can we maybe use 'readers' like in the rest of the style guide?

Reply:
Done :)